tools/azsdk-cli/Azure.Sdk.Tools.Cli/Tools/CustomizedCodeUpdateTool/README.md

```diff
@@ -0,0 +1,81 @@
+# TspClientUpdate Tool
+
+This tool provides a CLI and MCP entrypoint to automate updating customized client code when TypeSpec-generated code changes. It is language-agnostic — the system supports all languages via pluggable `IUpdateLanguageService` implementations.
+
+---
+
+## Overview
+
+This tool runs the `customized-update` workflow: regenerate TypeSpec output, diff old vs new generated code, map API changes to customization files, propose patches, and (optionally) apply them.
+
+---
+
+## Quick CLI examples
+
+Run from the repository root (uses the `Azure.Sdk.Tools.Cli` project):
+
+```powershell
+# Full workflow: regenerate -> diff -> map -> propose -> apply (dry-run)
+dotnet run --project Azure.Sdk.Tools.Cli/Azure.Sdk.Tools.Cli.csproj customized-update path/to/spec.tsp --new-gen ./tmpgen
```

Comment:
```suggestion
dotnet run --project Azure.Sdk.Tools.Cli/Azure.Sdk.Tools.Cli.csproj -- tsp customized-update path/to/spec.tsp --new-gen ./tmpgen
```

Not sure about the command group but any flags for the program need to be after `--`